.github/workflows/migrations.yml

```diff
@@ -0,0 +1,21 @@
+name: commcare-hq migrations check
+on:
+  pull_request:
+    branches:
+    - master
+    - hotfix-deploy
+jobs:
+  check_migrations:
+    name: Check migrations
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Verify up-to-date migrations lockfile
+      run: |
+        make migrations
```

Comment:
Yeah, this won't work I don't think we actually setup any env in the "workflow image". I need to add a "runner" inside the django docker container, spin up that container and kick it off. I think it will be something like:

```yaml
    - name: Test migrations
      env:
        TEST: migrations
        JS_SETUP: no
      run: scripts/docker test
```

Reply:
I do really like having this as a separate workflow though rather than part of the 'test' workflow